katana/wui/core/templates/core/index.html

```diff
@@ -0,0 +1,122 @@
+<html>
```

Comment:
Why has this file been modified/created?

Reply:
This was not modified by me. This came in with the merge of Suraj's branch.